README.md

```diff
@@ -90,6 +90,10 @@ The formats of these are the same as in the `mysql_user` module.
     mysql_socket: *default value depends on OS*
     mysql_pid_file: *default value depends on OS*
 
+For some recent versions of MariaDB you might not want to specify `pid-file`. For such situations, set a blank value.
```

Comment:
To further connect this line with the variable below, please do `set a blank value:` (instead of the period).